e2e/nomostest/config_sync.go

```diff
@@ -824,10 +824,15 @@ func RepoSyncObjectV1Beta1(nn types.NamespacedName, repoURL string, sourceFormat
 
 // RootSyncObjectOCI returns a RootSync object that syncs the provided OCIImage.
 func (nt *NT) RootSyncObjectOCI(name string, image *registryproviders.OCIImage) *v1beta1.RootSync {
+	imageURL, err := image.RemoteAddressWithTag()
+	if err != nil {
+		nt.T.Fatalf("OCIImage.SyncURLWithTag: %v", err)
```

Comment:
nit: `RemoteAddressWithTag` here and below

Reply:
fixed